server/src/main/java/io/seata/server/coordinator/DefaultCore.java

```diff
@@ -88,6 +88,7 @@ public Long branchRegister(BranchType branchType, String resourceId, String clie
                 globalSession.addBranch(branchSession);
             } catch (RuntimeException ex) {
                 branchSession.unlock();
+                LOGGER.error("Failed to store branch xid = {} branchId = {}", globalSession.getXid(), branchSession.getBranchId(), ex);
```

Comment:
Why don't you throw the exception details？

Reply:
fixed.